app/black_spots/views.py

```diff
@@ -78,14 +81,12 @@ def get_queryset(self):
 
 
 class EnforcerAssignmentViewSet(drf_mixins.ListModelMixin, viewsets.GenericViewSet):
-    """
-    ViewSet for enforcer assignments
+    """ViewSet for enforcer assignments"""
+    # Enforcer assignments do not currently have an associated model in the database, and are
```

Comment:
Out of curiosity, what was the reasoning behind switching from a docstring to inline comments?